fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/SessionFactory.java

```diff
@@ -136,6 +136,16 @@ public Session getSession(final HttpServletRequest servletRequest) {
     }
 
     /**
+     * Determine whether the given Session is part of a transaction
+     *
+     * @param session
+     * @return whether the session is part of a transaction
+     */
+    public boolean isInTransaction(final Session session) {
```

Comment:
I would like to remove JCR from the HTTP layers completely, but this PR is specifically about chipping away at the Modeshape dependency in the `-http-commons` and `-http-api` modules. Removing JCR is a separate (though very much related) thing.